core/src/plugins/outputs/index.ts

```diff
@@ -109,7 +114,8 @@ export default definePlugin(id, () => {
                     {
                       name: model.uri.toString(),
                       writeByteOrderMark: false,
-                      text: originalText
+                      text: originalText,
+                      tsCompilerResultText: ''
```

Comment:
这个地方其实是因为。。。。tsc 没办法编译 js，我看了一下 ts playground 的源码，他们也没有进行这个处理。
所以我们也需要将 tsCompilerResultText 也设置为 originalText。
```suggestion
                      tsCompilerResultText: originalText
```